Gems/PhysX/Code/Include/PhysX/Joint/PhysXJointBus.h

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+
+#pragma once
+
+#include <AzCore/Component/EntityId.h>
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/Interface/Interface.h>
+#include <AzCore/std/utils.h>
+#include <AzCore/Math/Transform.h>
+
+namespace PhysX
+{
+    //! Interface to communicate with PhysX joint's motor.
+    class JointInterface : public AZ::EBusTraits
+    {
+    public:
+        static constexpr AZ::EBusAddressPolicy AddressPolicy = AZ::EBusAddressPolicy::ById;
+        using BusIdType = AZ::EntityId;
```

Comment:
question: It might be worth double checking if we want this to be addressed by EntityComponentIdPair instead of EntityId (this would make it possible to have more than one joint on an entity if we wanted that). That might not be something that comes up but I just wanted to mention it in case.

Reply:
Changed to address AZ::EntityComponentIdPair.